src/testdrive/action/sql.rs

```diff
@@ -26,24 +25,28 @@ use ore::collections::CollectionExt;
 use pgrepr::{Interval, Numeric};
 
 use crate::action::{Action, State};
-use crate::parser::{FailSqlCommand, SqlCommand};
+use crate::parser::{FailSqlCommand, SqlCommand, SqlExpectedResult};
 
 pub struct SqlAction {
     cmd: SqlCommand,
     stmt: Statement,
+    timeout: u64,
```

Comment:
As mentioned in the previous file, I think it is cleaner to make this a `Duration`.